Gemfile

```diff
@@ -3,5 +3,3 @@
 source "https://rubygems.org"
```

Comment:
Since it doesn't look like Ruby's a dependency anymore, can we remove the Gemfile and lock entirely?

Reply:
Oh! Good point, thanks!